### ISSUE TITLE
Overseerr/Sonarr/Radarr roor folder

### ISSUE THREAD

**Issue by PierreDurrr** [Casvt/Plex-scripts#87]
OK, an hard one here...
Since Overseerr doesn't allow per user root folder, would it be possible, once a request is made by a predefined user or group of users to change sonarr/radarr root folder ?

**Comment by Casvt** [comment]
Yeah so "if user1 requested a movie/show, then set root folder inside radarr/sonarr to root1".
Yeah sure quite easy actually

**Comment by PierreDurrr** [comment]
> Yeah so "if user1 requested a movie/show, then set root folder inside radarr/sonarr to root1". Yeah sure quite easy actually

Yup that's totally it
